test/logger.test.js

```diff
@@ -191,14 +195,52 @@ describe('logger', () => {
             });
             sinon.spy(logger, '_createBulk');
 
+            logger.log('hello there from test');
```

Comment:
This `it` has 2 different tests
* Not adding nano seconds
* Adding nano seconds
Please split it

Reply:
The string should be a var so you wouldn't have to maintain it in multiple places